tema_3/main.py

```diff
@@ -0,0 +1,70 @@
+import json
+# import random
+import csv
+import os
+
+# generate a folder output_data with multiple .json files
+output_folder = "output_folder"
+if not os.path.exists(output_folder):
```

Comment:
Please do not run code directly in the module. Check if the module name is `__main__`.